doc/man/scons.xml

```diff
@@ -7236,33 +7232,39 @@ to run &scons;.</para>
 <refsect2 id='windows_sconsbat_file'>
 <title>Windows: <filename>scons.bat</filename> file</title>
 
-<para>On Windows systems,
-&scons; is executed via a wrapper
-<filename>scons.bat</filename>
-file.
-This has (at least) two ramifications:</para>
+<para>On Windows, if &scons; is executed via a wrapper
+<filename>scons.bat</filename> batch file,
```

Comment:
no more scons.bat.. right?

Reply:
One still exists if you use a local-install scons, since no .exe is produced in that case.  Whether anybody should use same is a question I don't know the answer to.

Reply:
ahh.. o.k. good point.